CHANGELOG.md

```diff
@@ -45,19 +57,20 @@ instructions below to upgrade your Postgres.
   * Track `cortex_querier_blocks_sync_seconds` metric for the initial sync too
   * Fixed race condition
```

Comment:
If you want this to be of value to a reader, say more about how they might have hit the race.
Or delete this line.

Reply:
Right. It's not of a real value, given it's an experimental feature. Removed.